src/python/notation.py

```diff
@@ -639,7 +639,7 @@
   ),
   ("2025-01-23", "Add a bias towards bit set encoding. https://github.com/apache/lucene/pull/14155", "Add a bias towards bit set encoding. https://github.com/apache/lucene/pull/14155"),
   (
-    "2025-01-26",
+    "2025-01-25",
```

Comment:
For reference, #14164 was merged on 2025-01-25. 

https://benchmarks.mikemccandless.com/2025.01.25.18.04.51.html